### ISSUE TITLE
.aac url not playing properly

### ISSUE THREAD

**Issue by ibigfire** [sedmelluq/lavaplayer#37]
Hello,
I see from the description that the player can play .aac files, which is fantastic, but for some reason http://ckpk.streamon.fm:8000/CKPK-192k.aac does not seem to work.  Is there anything that can be done to make it compatible?

Thank you.

**Comment by Frederikam** [comment]
I just tried proping that with ffprobe. Maybe the stream is malformed?

```
[http @ 0000000002063c80] Stream ends prematurely at 0, should be 18446744073709551615
http://ckpk.streamon.fm:8000/CKPK-192k.aac: I/O error```

**Comment by ibigfire** [comment]
That's a little beyond my expertise level to know, unfortunately.  I do know it plays just fine in Firefox and MusicBee.

The only other information I have that may be helpful is that anytime I start listening to it, it plays a short few second commercial before getting to the music.  I'm thinking maybe that is screwing it up, since it's doing something fancy there?  I tried using FF's inspector network monitor to see what I could see, and all I could tell is that it does give me a StreamOnPermID cookie thing, that maybe is allowing me to play it properly?

I don't know.  Sorry if the things I'm saying are dumb and irrelevant.  I'm a bit out of my league here.

**Comment by sedmelluq** [comment]
Fixed as of `1.2.50`.
